src/Npgsql/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 ﻿#nullable enable
+override Npgsql.NpgsqlBatch.Dispose() -> void
```

Comment:
It's silly that this is flagged as a public API change

Reply:
Yeah, it is 😢